docs/third-party-packages.md

```diff
@@ -238,3 +238,11 @@ An extension to integrate Swagger/OpenAPI document easily for Starlette project
 Provides translations, localization, and timezone support via Babel integration.
 
 <a href="https://github.com/alex-oleshkevich/starlette_babel" target="_blank">GitHub</a>
+
+### Sentry-Python
+
+<a href="https://github.com/getsentry/sentry-python" target="_blank">GitHub</a> | <a href="https://docs.sentry.io/platforms/python/guides/starlette/" target="_blank">Documentation</a>
+
+Sentry is a tool that finds and reports errors in software. The Sentry Python SDK is what makes it work with Python. It also gives actionable insights to resolve Python performance bottlenecks and errors. One can see the full picture of any Python exception so you can diagnose, fix, and optimize performance in the Python debugging process. Sentry has dedicated integration with Starlette for building applications in Python. Sentry can help with Error tracking; performance insights; contextual information and Alerts and notifications.
+
+Powered by <a href="https://sentry.io/" target="_blank">sentry.io</a>
```

Comment:
I think this last line is very confusing.
Looks like the docs page is powered by Sentry 😅

Reply:
ya, now if i look back it sure seems that way. I have updated it by adding a website tag. Hope that's good.